examples/simple-platform/user-walk-through.md

```diff
@@ -0,0 +1,49 @@
+# User Walk-through of the Platform Example
+
+## Creating an account and going through the GitHub OAuth loop
+
+Open the accounts app in your browser at `http://localhost:3025`. If you open 
+your browser's developer tools to the console tab, you can see some 
+`interbit-middleware` log messages being printed. The accounts app is fully up 
+and running once you see the `interbit-middleware/CHAIN_BLOCKING` status for 
+the `accountsPrivate` chain. At this point, you will be able to dispatch 
+actions to your private account chain.  
+
+From the main accounts app page, click on the **Create Account / Sign in** 
+link in the header, which navigates you to the `create-account` page. Click 
+the **Create Account** button, check that you have read the disclaimer and 
+click **Continue** in the modal.
+
+If you're already signed into your GitHub account, you will be directed to the 
+your personal account page. Otherwise, you will be redirected to GitHub and 
+prompted to enter your GitHub credentials. Enter your credentials and sign in. 
+You should be redirected back to the accounts app. 
+
+You will see a number of chain actions being dispatched in the browser console 
+on the accounts app tab. Specifically, after the action type 
+`app-account-my-account/COMPLETE_AUTHENTICATION` is dispatched, the 
+`chainState.profile` property is updated with your GitHub identity 
+information. You can also verify that the OAuth loop was successful if you see 
+your GitHub username appear in the header. 
+
+Congratulations, you have now created an Interbit account. You can make 
+changes to your personal account information and see how the chain blocks with 
```

Comment:
What is meant by `blocks`? It's terminology common to experienced blockchain developers. Readers of this example need to know that you mean "new blocks added to the blockchain", as opposed to "prevents other actions from taking place".